packages/ckeditor5-dev-tests/lib/utils/manual-tests/compilehtmlfiles.js

```diff
@@ -22,8 +22,10 @@ const writer = new commonmark.HtmlRenderer();
 const viewTemplate = fs.readFileSync( path.join( __dirname, 'template.html' ), 'utf-8' );
 
 module.exports = function compileManualTestHtmlFiles( buildDir, manualTestPattern ) {
-	const sourceMDFiles = globSync( path.join( manualTestPattern, '*.md' ) );
-	const sourceDirs = uniq( sourceMDFiles.map( file => path.dirname( file ) ) );
+	const sourceMdFiles = globSync( path.join( manualTestPattern, '*.md' ) );
```

Comment:
https://github.com/ckeditor/ckeditor5-design/wiki/Code-Style-Naming-Guidelines#acronyms-and-proper-names

> However, two letter acronyms and proper names (if originally written uppercase) should be uppercase. So e.g. getUI (not getUi).

Longer acronyms should be `Xxxx`.